Add VPC Classic IDE schedule disclaimer (#2607)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/03-Nov-2022/ide-features-ide-deprecation.md

In December 2022, dbt Labs will deprecate the classic IDE. The [new and refreshed IDE](/docs/get-started/develop-in-the-cloud) will be available for _all_ dbt Cloud users. You will no longer be able to access the classic IDE and dbt Labs might introduce changes that break the classic IDE.

With deprecation, dbt Labs will only support the refreshed version of the dbt Cloud IDE.

> Virtual Private Cloud (VPC) customers with questions about when this change will affect their account, should check with their account team or support contact.

Reviewer: ```suggestion
Virtual Private Cloud (VPC) customers with questions about when this change will affect your account can contact your account team or support contact for assistance.
```
